python/paddle/fluid/layers/nn.py

```diff
@@ -6072,6 +6073,127 @@ def crop(x, shape=None, offsets=None, name=None):
     return out
 
 
+def affine_grid_generator(theta, out_shape, name=None):
+    """
+    It generates a grid of (x,y) coordinates using the parameters of
+    the affine transformation that correspond to a set of points where
+    the input feature map should be sampled to produce the transformed
+    output feature map.
+
+
+    Crop input into output, as specified by offsets and shape.
+
+    .. code-block:: text
+
+        * Case 1:
+
+          Given:
+
+              theta = [[[x_11, x_12, x_13]
+                        [x_14, x_15, x_16]]
+                       [[x_21, x_22, x_23]
+                        [x_24, x_25, x_26]]]
+      
+              out_shape = [2, 3, 5, 5]
+      
+          Step 1:
+      
+              Generate relative coordinates according to Size.
```

Comment:
what is `Size`?

Reply:
Rename `size` to `out_shape` which is the shape of target output with format [N, C, H, W]